### PR TITLE
feat(words): add new vocab for the intrinsic camera calibrator

### PR DESCRIPTION
**Pull request by knzo25**
Signed-off-by: Kenzo Lobos-Tsunekawa <kenzo.lobos@tier4.jp>

### CODE REVIEW

**Comment by kenji-miyake** [comment]
@knzo25 @aohsato Could you show me an example SVG file?

**Comment by kenji-miyake** [comment]
Or are these not for SVG files? :thinking:

**Comment by knzo25** [comment]
SVG files?
these are new words to a tool to calibrate the camera intrinsics

**Comment by kenji-miyake** [comment]
At first, the words were added to the SVG section.
image

But now it looks to be okay!

**Comment by kenji-miyake** [comment]
I'll fix the title again.

File: .cspell.json [hunk 1]
```diff
@@ -282,7 +282,9 @@
     "appinfo",
     "Applanix",
     "apriltag",
+    "apriltags",
     "arange",
+    "arccos",
```
Comment (kenji-miyake) [inline .cspell.json:287]: How about also adding related words?

```suggestion
    "arccos",
    "arcsin",
    "arctan",
```

Comment (kenji-miyake) [reply .cspell.json:287]: Oh, `arctan` was already there.

**Comment by knzo25** [comment]
We switched to our custom dict so I will be closing this one :bow:

**Pull request closed without merging**